independent-projects/enforcer-rules/pom.xml

```diff
@@ -40,6 +40,10 @@
         <maven-core.version>3.8.6</maven-core.version>
 
         <nexus-staging-maven-plugin.version>1.6.8</nexus-staging-maven-plugin.version>
+        <!--
+           Supported Maven versions, interpreted as a version range (Also defined in build-parent)
+        -->
+        <supported-maven-versions>[3.6.2,)</supported-maven-versions>
```

Comment:
I'm not sure it makes sense to define this one here. Rationale: the supported Maven version really depends on the project you are currently building and Quarkus has nothing to do with it.

Reply:
I'd be interested to get `<supported-maven-versions>` in Camel Quarkus. I think there will inevitably be entries in this rules file that we won't be wanting to enforce. A XSLT pre-filtering will solve all such issues.

Reply:
What if we split the rules into several files (one for each rule type)? 
e.g. 
- `quarkus-require-java-version.xml`;
- `quarkus-require-maven-version.xml`;
- `quarkus-banned-dependencies.xml`

Reply:
> What if we split the rules into several files

Hm... how do I get notified that there is a new rule file?

Reply:
We could still have a `quarkus-default.xml` with <enforcerRules> to each of these individual files which if you use it you don't need to bother about new rule files.